src/SerialLoops/ProjectCreationDialog.eto.cs

```diff
@@ -25,50 +25,95 @@ partial class ProjectCreationDialog : Dialog
         void InitializeComponent()
 		{
 			Title = "Create New Project";
-			MinimumSize = new Size(400, 300);
+			MinimumSize = new Size(400, 275);
 			Padding = 10;
 
-            _nameBox = new();
+			_nameBox = new()
+			{
+				PlaceholderText = "haroohie",
+				Size = new Size(150, 25)
+			};
 			_languageDropDown = new();
 			_languageDropDown.Items.AddRange(_availableLanguages.Select(a => new ListItem() { Text = a.Key, Key = a.Value }));
+			_languageDropDown.SelectedIndex = 0;
 			_romPath = new() { Text = NO_ROM_TEXT };
 			Command pickRomCommand = new();
             pickRomCommand.Executed += PickRomCommand_Executed;
             Command createCommand = new();
             createCommand.Executed += CreateCommand_Executed;
+            Command cancelCommand = new();
+            cancelCommand.Executed += CancelCommand_Executed;
 
 			Content = new StackLayout
 			{
+				Spacing = 10,
+				HorizontalContentAlignment = HorizontalAlignment.Center,
 				Items =
 				{
-					new StackLayout
+					new GroupBox
 					{
-						Orientation = Orientation.Horizontal,
-						Items =
+						Text = "Project Options",
+						Padding = 5,
+						MinimumSize = new Size(300, 100),
+						Content = new StackLayout
 						{
-							"Name: ",
-							_nameBox,
+							Spacing = 10,
+							VerticalContentAlignment = VerticalAlignment.Center,
+							Items = { new StackLayout
```

Comment:
Nit, but indentation here is a little wonky, I'd do:
```suggestion
							Items =
							{
							    new StackLayout
```